### PR TITLE
server: disable documentation by default

### PR DESCRIPTION
**Pull request by zimbatm**
Keep the evaluation and closure size small

### CODE REVIEW

File: nixos/server/default.nix [hunk 1]
```diff
@@ -17,6 +17,8 @@
     pkgs.vim
   ];
 
+  documentation.enable = lib.mkDefault false;
```
Comment (Mic92) [inline nixos/server/default.nix:20]:
```suggestion
  # Notice this also disables --help for some commands such es nixos-rebuild
  documentation.enable = lib.mkDefault false;
```

Comment (Mic92) [reply nixos/server/default.nix:20]: I had the idea to turn this into a standalone package:

```
$ nix repl
Welcome to Nix 2.13.3. Type :? for help.

nix-repl> :lf .#                                                         
Added 19 variables.

nix-repl> :b nixosConfigurations.eve.config.system.build.manual.manpages

This derivation produced the following outputs:
  out -> /nix/store/919zbm49lcizzdd66myyrgwzkb8fm444-nixos-manpages

nix-repl> 
```

similar to nix-index-database.

**Comment by zimbatm** [comment]
bors merge

File: nixos/server/default.nix (hunk 1 above)
Comment (Mic92) [reply nixos/server/default.nix:21]: I started some repo, will update you once I got it running.

**Pull request merged**